webpack.config.js

```diff
@@ -1,14 +1,18 @@
-var path = require('path');
-var webpack = require('webpack');
+var path = require('path')
+var webpack = require('webpack')
 
 module.exports = {
     entry: {
-        "batavia": "./batavia/batavia.js",
+        'batavia': './batavia/batavia.js'
+    },
+    watchOptions: {
+      aggregateTimeout: 300,
+      poll: 1000
     },
     devtool: 'source-map',
     output: {
         path: path.join(__dirname, './dist'),
```

Comment:
At column 1: [(indent) Expected indentation of 8 spaces but found 6.](http://eslint.org/docs/rules/indent)